model_hub/examples/huggingface/translation/translation_trial.py

```diff
@@ -0,0 +1,348 @@
+"""
+This example is largely based on the translation example in the huggingface
+transformers library. The license for the transformer's library is reproduced below.
+
+==================================================================================================
+
+Copyright The HuggingFace Team and The HuggingFace Inc. team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import functools
+import logging
+from typing import Any, Dict, Tuple, Union
+
+import attrdict
+import datasets
+import numpy as np
+import torch
+import transformers
+
+import determined.pytorch as det_torch
+import model_hub.huggingface as hf
+import model_hub.utils as utils
+
+# A list of all multilingual tokenizer which require src_lang and tgt_lang attributes.
+MULTILINGUAL_TOKENIZERS = [
+    transformers.MBartTokenizer,
+    transformers.MBartTokenizerFast,
+    transformers.MBart50Tokenizer,
+    transformers.MBart50TokenizerFast,
+    transformers.M2M100Tokenizer,
+]
+
+
+class TranslationTrial(hf.BaseTransformerTrial):
```

Comment:
One thing I noticed that could be helpful here is a link to the docs so people can look at some of the `model_hub.utils` functions that you use here in case they are confused where that comes from. Ditto the other model_hub examples as well.

Reply:
good suggestion, i added a reference for each of the transformer trials